articles/cosmos-db/partition-data.md

```diff
@@ -52,10 +52,10 @@ The semantics for partition keys are slightly different to match the semantics o
 | Graph | Custom partition key property | Fixed `id` | 
 | Table | Fixed `PartitionKey` | Fixed `RowKey` | 
 
-Azure Cosmos DB uses hash-based partitioning. When you write an item, Azure Cosmos DB hashes the partition key value and uses the hashed result to determine which partition to store the item in. Azure Cosmos DB stores all items with the same partition key in the same physical partition. The choice of the partition key is an important decision that you have to make at design time. You must pick a property name that has a wide range of values and has even access patterns.
+The choice of the partition key is an important decision that you have to make at design time. You must pick a property name that has a wide range of values and has even access patterns.
 
 > [!NOTE]
-> It's a best practice to have a partition key with many distinct values (hundreds to thousands at a minimum).
+> It's a best practice to have a partition key with many distinct values (hundreds to thousands at a minimum). Remember! partition key is a logical concept and partitions are physical partitions. These are two different thing. For a 10 partition system, you can have tens of thousands of partition keys.
```

Comment:
Please change to "Remember, the partition key is a logical ...", we can't use ! mid sentence. Or we can turn it into a NOTE, or TIP, or IMPORTANT item, which will put it in a colored box on the screen. Let me know your preference.